controller/namedspaces_test.go

```diff
@@ -120,9 +122,125 @@ func (rest *TestNamedSpaceREST) TestSuccessListSpaces() {
 	assert.NotNil(t, created.Data.Links.Self)
 
 	collabSpaceSvc, collabSpacectrl := rest.SecuredNamedSpaceController(testsupport.TestIdentity)
-	_, collabspaces := test.ListNamedspacesOK(t, collabSpaceSvc.Context, collabSpaceSvc, collabSpacectrl, testsupport.TestIdentity.Username, nil, nil)
+	_, collabspaces := test.ListNamedspacesOK(t, collabSpaceSvc.Context, collabSpaceSvc, collabSpacectrl, testsupport.TestIdentity.Username, nil, nil, nil)
 	assert.True(t, len(collabspaces.Data) > 0)
 	assert.Equal(t, created.Data.Attributes.Name, collabspaces.Data[0].Attributes.Name)
 	assert.Equal(t, created.Data.Attributes.Description, collabspaces.Data[0].Attributes.Description)
 	assert.Equal(t, created.Data.Links.Self, collabspaces.Data[0].Links.Self)
 }
+
+func (rest *TestNamedSpaceREST) TestSuccessSortListSpaces() {
+	t := rest.T()
+	resource.Require(t, resource.Database)
+
+	spaceSvc, spaceCtrl := rest.SecuredSpaceController()
+	identityRepo := account.NewIdentityRepository(rest.DB)
+	identity := testsupport.TestIdentity
+	identity.ProviderType = account.KeycloakIDP
+	err := identityRepo.Create(spaceSvc.Context, &identity)
+	if err != nil {
+		assert.Fail(t, "Failed to create an identity")
+	}
+
+	// create spaces
+	inputSpaces := []string{
+		"Test 24",
+		"Test 22",
+		"Test 23",
+	}
+	// randominze the space names so that they don't clash
+	random := testsupport.CreateRandomValidTestName("")
+	for i := range inputSpaces {
+		inputSpaces[i] += random
+	}
+
+	for _, sn := range inputSpaces {
+		p := newCreateSpacePayload(&sn, nil)
+		test.CreateSpaceCreated(t, spaceSvc.Context, spaceSvc, spaceCtrl, p)
+	}
```

Comment:
Can you please create the spaces like this?

```go
fxt := tf.NewTestFixture(t, s.DB, tf.Spaces(len(inputSpaces), func(fxt *tf.TestFixture, idx int) error {
			fxt.Spaces[idx].Name = testsupport.CreateRandomValidTestName(inputSpaces[idx])
			return nil
		}),
)
```

It would greatly help moving the code forward.

Reply:
done